src/metabase/search/appdb/scoring.clj

```diff
@@ -152,7 +163,12 @@
    :recency      (inverse-duration [:coalesce :last_viewed_at :model_updated_at] [:now] search.config/stale-time-in-days)
    :user-recency (inverse-duration (user-recency-expr search-ctx) [:now] search.config/stale-time-in-days)
    :dashboard    (size :dashboardcard_count search.config/dashboard-count-ceiling)
-   :model        (model-rank-exp search-ctx)})
+   :model        (model-rank-exp search-ctx)
+   :mine         (equal :search_index.creator_id (:current-user-id search-ctx))
+   :exact        (equal [:lower :search_index.name] [:lower (:search-string search-ctx)])
+   :prefix       (if (= :command-palette (:context search-ctx))
+                   (prefix [:lower :search_index.name] (u/lower-case-en (:search-string search-ctx)))
+                   [:inline 0])})
```

Comment:
I realize now that using a contextual weight is a better route, going to update it.